templates/emails/plain/failed-refund-customer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+echo esc_html( $email_heading ) . "\n\n";
+
+printf(
+	// translators: 1) an order number, 2) the reason for the failure.
+	esc_html_x(
+		'The refund request for order %1$s has failed. Reason: %2$s.',
+		'In customer refund failed email',
+		'woocommerce-gateway-stripe'
+	),
+	esc_html( $order->get_order_number() ),
+	esc_html( $reason )
+) . "\n\n";
+printf( esc_html__( 'The refunded order is as follows:', 'woocommerce-gateway-stripe' ) ) . "\n\n";
```

Comment:
I think we should change the wording here. "The refunded order" implies that the order was refunded. I think we should just use "The original order" or "The order details are as follows:".